tests/Integration/RequestIntegrationTest.php

```diff
@@ -304,7 +304,8 @@ public function invokeDataProvider(): iterable
             (new ServerRequest('POST', new Uri('http://www.example.com/foo')))
                 ->withUploadedFiles([
                     'foo' => new UploadedFile('foo content', 123, UPLOAD_ERR_OK, 'foo.ext', 'application/text'),
-                ]),
+                ])
+                ->withBody($this->getStreamMock(123 + 321)),
```

Comment:
What's the reason of adding `withBody()` to all test cases? The only change I would expect is the new test case that asserts that when the `$requestBodySize` is 0 then everything works fine

Reply:
I realized only now that it's needed because the handling of the uploaded files happen only if the stream size is greater than 0, so this is fine